protoactor/dispatcher.py

```diff
@@ -14,18 +15,26 @@ def throughput(self) -> int:
     def schedule(self, runner: Callable[..., asyncio.Task]):
         raise NotImplementedError("Should Implement this method")
 
+def _run_async(runner):
+    async_loop = asyncio.new_event_loop()
+    # task = asyncio.wait(async_loop.create_task(runner()))
+    async_loop.run_until_complete(runner())
+    async_loop.close()
 
 class ProcessDispatcher(AbstractDispatcher):
     @property
     def throughput(self) -> int:
-        raise NotImplementedError("Should Implement this method")
+        return 300
 
     def schedule(self, runner: Callable[..., asyncio.Task]):
-        def run_async(runner):
-            async_loop = asyncio.get_event_loop()
-            task = asyncio.wait(async_loop.create_task(runner()))
-            async_loop.run_until_complete(task)
-            async_loop.close()
-
-        p = Process(target=run_async, args=runner)
+        p = Process(target=_run_async, args=(runner,))
         p.start()
+
+class ThreadDispatcher(AbstractDispatcher):
```

Comment:
The idea is to offer different dispatchers due Python nature one might me more appropriate in certain cases than others